pom.xml

```diff
@@ -52,67 +49,80 @@
     </dependency>
   </dependencies>
+
+  <repositories>
+    <repository>
+      <releases>
+        <enabled>false</enabled>
+      </releases>
+      <snapshots>
+        <enabled>true</enabled>
+      </snapshots>
+      <id>jetty.snapshots</id>
+      <url>https://oss.sonatype.org/content/repositories/jetty-snapshots</url>
+    </repository>
+  </repositories>
+
+  <pluginRepositories>
+    <pluginRepository>
+      <releases>
+        <enabled>false</enabled>
+      </releases>
+      <snapshots>
+        <enabled>true</enabled>
+      </snapshots>
+      <id>jetty.snapshots</id>
+      <url>https://oss.sonatype.org/content/repositories/jetty-snapshots</url>
+    </pluginRepository>
+  </pluginRepositories>
   <build>
+    <pluginManagement>
+      <plugins>
+        <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-compiler-plugin</artifactId>
+          <configuration>
+            <compilerArgument>-Xlint:all</compilerArgument>
+          </configuration>
+        </plugin>
+      </plugins>
+    </pluginManagement>
     <plugins>
-      <plugin>
-        <artifactId>maven-compiler-plugin</artifactId>
-        <version>3.11.0</version>
-        <configuration>
-          <source>${compiler.source}</source>
-          <target>${compiler.target}</target>
-          <release>${compiler.release}</release>
-          <showWarnings>true</showWarnings>
-          <compilerArgument>-Xlint:all</compilerArgument>
-        </configuration>
-      </plugin>
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
-        <artifactId>maven-release-plugin</artifactId>
-        <version>3.0.1</version>
-        <configuration>
-          <useReleaseProfile>true</useReleaseProfile>
-          <releaseProfiles>eclipse-release</releaseProfiles>
-          <autoVersionSubmodules>true</autoVersionSubmodules>
-        </configuration>
-      </plugin>
-      <plugin>
-        <groupId>org.apache.maven.plugins</groupId>
-        <artifactId>maven-enforcer-plugin</artifactId>
-        <version>3.4.1</version>
+        <artifactId>maven-assembly-plugin</artifactId>
+        <dependencies>
+          <dependency>
+            <groupId>org.eclipse.jetty.toolchain</groupId>
+            <artifactId>jetty-assembly-descriptors</artifactId>
+            <version>1.1</version>
+          </dependency>
+        </dependencies>
         <executions>
           <execution>
-            <id>enforce-build-env</id>
+            <id>assemble-config</id>
             <goals>
-              <goal>enforce</goal>
+              <goal>single</goal>
             </goals>
-            <phase>validate</phase>
+            <phase>package</phase>
             <configuration>
-              <rules>
-                <requireMavenVersion>
-                  <version>[3.8.4,)</version>
-                  <message>[ERROR] OLD MAVEN [${maven.version}] in use, Jetty ${project.version} requires Maven 3.8.4 or newer</message>
```

Comment:
This enforcer configuration NEEDS to be at the child repo level.
Not in a parent pom.

Reply:
it's using a property so you can override it